.travis.yml

```diff
@@ -30,8 +30,9 @@ install:
   - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && ./gradlew :ui:assemble --stacktrace || ./gradlew --stacktrace '
 
 script:
+  - if [[ "TRAVIS_OS_NAME" == "osx" ]]; then ./gradlew :ui:test -Pgeneration; else
```

Comment:
Make the whole `if-else-fi` block multiline. It's hard to understand when it has inconsistent formatting.